src/e2e-test/java/io/cdap/cdap/ui/utils/Constants.java

```diff
@@ -67,4 +67,5 @@ private static double getRandomArbitrary(int min, int max) {
 
   public static final String DEFAULT_GCP_PROJECTID = System.getenv("GCP_PROJECTID");
   public static final String DEFAULT_GCP_SERVICEACCOUNT_PATH = System.getenv("GCP_SERVICE_ACCOUNT_PATH");
+  public static final String WRANGLE_HOME_URL = "http://localhost:11011/cdap/ns/default/home";
```

Comment:
This should be built on top of `BASE_URL`

Reply:
Done